src/main/java/Duke/Task.java

```diff
@@ -0,0 +1,108 @@
+package Duke;
+
+/**
+ * Duke.Task class to store the states whether task is complete or not
+ */
+public class Task {
+    /**
+     * boolean state to store completed or not.
+     */
+    private boolean completed;
+
+    /**
+     * name of task.
+     */
+    private final String taskName;
+
+
+    /**
+     * Constructor to initialize class.
+     *
+     * @param name task name
+     */
+    public Task(String name, boolean completed) {
+        taskName = name;
+        this.completed = completed;
+    }
+
+    /**
+     * Sets complete to true.
+     */
+    public void mark() {
+        completed = true;
+        System.out.printf(
+                "    ____________________________________________________________\n" +
```

Comment:
Likewise here, I think it would be good to abstract out the line break into a variable as well!